server/model/avg.py

```diff
@@ -0,0 +1,37 @@
+#import labels from model.GeminiClient
+from model.GeminiClient import POLITICAL_LABELS
+
+#political labels manually listed out
+#POLITICAL_LABELS = ["communist", "leftist", "liberal", "moderate", "conservative", "nationalist"]
+
+def avg_of_objects(dicts_of_json: dict) -> dict:

```

Comment:
can you try calling this function [here](https://github.com/chumomega/echo_chamber/blob/main/server/controllers/getEchoChamberStatus.py#L28) with the comments as input ?

then could you update the response here? 

https://github.com/chumomega/echo_chamber/blob/main/server/controllers/getEchoChamberStatus.py#L34